clustermesh-apiserver/clustermesh/health.go

```diff
@@ -50,6 +50,20 @@ func registerHealthAPIServer(lc hive.Lifecycle, clientset k8sClient.Clientset, c
 		}
 	})
 
+	mux.HandleFunc("/readyz", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Is the `readyz` endpoint supposed to replace the `healthz` one? Currently, `readyz` appears to be a startup check only, while `healthz` an (incomplete, as does not include etcd checks) runtime one. But `healthz` is never used.

Reply:
I think it makes sense to use `readyz` for readiness and extend `healthz` to cover etcd and use it for a livenessProbe. Does that sound good to you?

Reply:
I personally think it could be a good idea having a dedicated issue to discuss about the complete readiness/liveness probe strategy for the different clustermesh-apiserver containers. IMO the tricky part is finding a good balance to ensure that we have a good signal whether everything is working properly, without risking to make the situation even worse in case of temporary blips (e.g., restarting the clustermesh-apiserver would cause a full resynchronization, risking to overload etcd given that watches are already established). That said, it doesn't looks a blocker for this PR, which is already a nice improvement.